templates/common/page_start.html

```diff
@@ -18,6 +18,9 @@
   $else:
     <title>Weasyl</title>
 
+  <!-- If we have JavaScript enabled, remove the no-js class -->
```

Comment:
This could be a template comment to save sending some bytes, at least:

```suggestion
  $# If we have JavaScript enabled, remove the no-js class
```